arch/xtensa/core/xtensa_intr.c

```diff
@@ -57,13 +57,17 @@ xt_exc_handler _xt_set_exception_handler(int n, xt_exc_handler f)
 
 #endif
 
-#if XCHAL_HAVE_INTERRUPTS
-/*
- * Default handler for unhandled interrupts.
- */
-void xt_unhandled_interrupt(void *arg)
+#if defined(CONFIG_SW_ISR_TABLE) && defined(XCHAL_HAVE_INTERRUPTS)
+void _irq_spurious(void *arg)
 {
 	ReservedInterruptHandler((unsigned int)arg);
 	CODE_UNREACHABLE;
 }
-#endif /* XCHAL_HAVE_INTERRUPTS */
+
+struct _isr_table_entry __sw_isr_table _sw_isr_table[IRQ_TABLE_SIZE] = {
```

Comment:
shouldn't need this.
if gen_isr_tables.py is in use, it should build arch/common/isr_tables.c which creates the dummy table.